cohortextractor/tpp_backend.py

```diff
@@ -2788,6 +2824,15 @@ def patients_outpatient_appointment_date(
               OPA.Patient_ID
             """
 
+    @staticmethod
+    def _format_risk_group(original_string):
+        # First remove any "Patients with [a]" and replace " and " with "," within individual risk group fields
```

Comment:
Are the square brackets significant? (`[a]`)

Reply:
It's just shorthand for "Patients with" and "Patients with a" - on the forms there are checkboxes that list conditions as e.g. "Patients with a haematologic malignancy" and "Patients with renal disease"

Reply:
Ah, okay, I see. Thanks for clarifying!